Detectors/MUON/MCH/Workflow/CMakeLists.txt

```diff
@@ -34,6 +34,12 @@ o2_add_executable(
         COMPONENT_NAME mch
         PUBLIC_LINK_LIBRARIES O2::MCHWorkflow)
 
+o2_add_executable(
```

Comment:
if that device is only intended for debug, I would make it crystal clear by adding -debug- (or test or whatever) to the executable name

Reply:
I renamed the code and executable as `raw-debug-workflow`. Is that ok for you?